pkg/cli/cmd/configure.go

```diff
@@ -15,10 +15,11 @@ type ConfigureCmd struct {
 	PolicyName        string `short:"n" help:"policy name"`
 	LocalPolicyImage  string `short:"l" help:"local policy image name"`
 	Resource          string `short:"r" help:"resource url"`
-	Stdout            bool   `short:"p" help:"generated configuration is printed to stdout but not saved"`
+	Stdout            bool   `short:"p" help:"print to stdout"`
 	EdgeDirectory     bool   `short:"d" help:"enable edge directory" default:"false"`
-	Force             bool   `flag:"" default:"false" short:"f" help:"forcefully create configuration"`
-	EnableDirectoryV2 bool   `flag:"" name:"enable-v2" default:"false" help:"enable directory version 2 services for backward compatibility"`
+	Force             bool   `flag:"" default:"false" short:"f" required:"false" help:"skip confirmation prompt"`
+	EnableDirectoryV2 bool   `flag:"" name:"enable-v2" hidden:"" default:"true" help:"enable directory version 2 services for backwards compatibility"`
+	ConfigFile        string `name:"config" json:"config,omitempty" default:"config.yaml" short:"c" help:"topaz configuration file"`
```

Comment:
What do you think about calling this field `ConfigName` and lose the `.yaml` extension.
When we read and write configs from/to disk we'll include the `.yaml` suffix but when users create, list, or use configurations, they'll just specify the file name.
For example, if I have a config file called `peoplefinder.yaml`, the configuration name would be just `peoplefinder`.

When calling `topaz configure` we could default to using the policy name as the config name, unless the caller explicitly sets the `ConfigName` param to override it.
So we could run
```bash
topaz configure -r ghcr.io/aserto-policies/policy-peoplfinder:latest -n peoplefinder
```
and generate `peoplefinder.yaml`.

Reply:
I'll put this change in a separate PR so it's easier to review